src/components/Select/types.ts

```diff
@@ -5,6 +5,8 @@ import {Option, OptionGroup} from './tech-components';
 
 export type SelectProps = Pick<ControlGroupProps, 'name' | 'disabled'> & {
     onUpdate?: (value: string[]) => void;
+    onOpenChange?: (args: {open: boolean}) => void;
```

Comment:
I`ve used object as function`s args to have flexibility in possible future improvements

Reply:
Not sure about that. We should do it either everywhere in the code or nowhere. There is exactly the same prop in Popover component. Callbacks should be consistent.